gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -321,6 +322,11 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
           "list of accelerators determines the backend (ignored with false). "
           "Example, if GPU, NPU can be used but not CPU - true:(GPU,NPU,!CPU).",
           "", G_PARAM_READWRITE));
+  g_object_class_install_property (gobject_class, PROP_UPDATEMODEL,
+      g_param_spec_string ("updatemodel", "Updatable model",
```

Comment:
how about changing property name: `is- updatable`

Reply:
Looks good. thx.